app/javascript/components/custom-url-tabs/index.jsx

```diff
@@ -0,0 +1,99 @@
+import React, { useEffect, useState } from 'react';
+import PropTypes from 'prop-types';
+import { Tab, Tabs, Modal } from 'carbon-components-react';
+import { checkForFormChanges } from './helper';
+
+const CustomURLTabs = ({
+  tabs, path, currentTab, checkForChanges,
+}) => {
+  const [tabsList, setTabsList] = useState([]);
+  const [selectedTab, setSelectedTab] = useState(0);
+  const [showConfirm, setShowConfirm] = useState(false);
+  const [url, setUrl] = useState();
+
```

Comment:
this className seems to be repeated, so I guess we can use it as a const - 
`const selectedClass = 'bx--tabs--scrollable__nav-item--selected';`